docs: fix headers in "Getting Started" code-snippets

ajitsinghkaler opened the pull request:
In the your first app page there is a typo it was refering product-list folder but actuall it should refer the product-alert folder

Fixes #34320

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?
Small type changes in content instead of alert it was referring the list

<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
path is written wrong
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: 34320

## What is the new behavior?
Corrected the path

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

MrJithil commented on aio/content/start/index.md:
```diff
@@ -255,11 +255,11 @@ The next step is to create a new alert feature that takes a product as an input.
     1. Import `Input` from `@angular/core`.
 
-        <code-example path="getting-started/src/app/product-alerts/product-alerts.component.1.ts" region="imports" header="src/app/product-list/product-alerts.component.ts"></code-example>
+        <code-example path="getting-started/src/app/product-alerts/product-alerts.component.1.ts" region="imports" header="src/app/product-alerts/product-alerts.component.ts"></code-example>
```
Seems the associated issue need to elaborate further. Because, I could see both `src/app/product-list/product-alerts.component.ts` and `src/app/product-alerts/product-alerts.component.ts` exist.

gkalpak replied:
No, `product-list/product-alerts.component.ts` should not exist (if you've followed the steps correctly). See also https://github.com/angular/angular/issues/34320#issuecomment-563963216.

gkalpak reviewed (approved):
Thx, @ajitsinghkaler 👍
For the sake of clarity, could you please change the commit message to something like:

> docs: fix headers in "Getting Started" code-snippets
>
> The headers of two of the code-snippets in the \[Input section](https://angular.io/start#input)
> of the "Getting Started" guide incorrectly referenced an incorrect file
> path (\`src/app/product-list/product-alerts.component.ts\`).
>
> This commit fixes the headers to show the correct file path
> (\`src/app/product-alerts/product-alerts.component.ts\`).
>
> Fixes #34320

ajitsinghkaler committed:
docs: fix headers in "Getting Started" code-snippets

The headers of two of the code-snippets in the [Input section](https://angular.io/start#input)
of the "Getting Started" guide incorrectly referenced an non-existent
file path (`src/app/product-list/product-alerts.component.ts`).

This commit fixes the headers to show the correct file path
(`src/app/product-alerts/product-alerts.component.ts`).

Fixes #34320

ajitsinghkaler commented:
@gkalpak I made the changes

gkalpak reviewed (approved):
Thx, @ajitsinghkaler :+1:

gkalpak commented:
**merge-assistance:** Global approval for this docs-only change.
